Gems/Atom/Feature/Common/Code/Source/ImGui/ImGuiPass.h

```diff
@@ -143,12 +144,12 @@ namespace AZ
             Data::Instance<RPI::Shader> m_shader;
 
             Data::Instance<RPI::ShaderResourceGroup> m_resourceGroup;
-            RHI::ShaderInputNameIndex m_fontImageIndex = "FontImage";
+            RHI::ShaderInputNameIndex m_texturesIndex = "m_textures";
             RHI::ShaderInputNameIndex m_projectionMatrixIndex = "m_projectionMatrix";
             RHI::Viewport m_viewportState;
 
             RHI::IndexBufferView m_indexBufferView;
-            AZStd::array<RHI::StreamBufferView, 1> m_vertexBufferView; // Only 1 vertex stream view needed, but most RHI apis expect an array.
+            AZStd::array<RHI::StreamBufferView, 2> m_vertexBufferView; // For vertex buffer and instance data since most RHI apis expect an array.
```

Comment:
nit - Now that there's two items in there, the comment can probably just say `// For vertex buffer and instance data`

Reply:
Done!